tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv1-pages-questions.cy.js

```diff
@@ -0,0 +1,63 @@
+import DataMapper from "export-processor/data-mapper.js";
+import { selfAssessmentSlug } from "./helpers/index.js";
+import { validateNavigationLinks, validateNonAuthorisedPages, validateSections, validateSelfAssessmentPage } from "./validators/index.js";
```

Comment:
Is there a better possible name for the `validateSections` and `validateRecommendations` methods? They don't match the rest of the validators in terms of functionality/purpose.